packages/create-cloudflare/src/__tests__/common.test.ts

```diff
@@ -12,11 +13,21 @@ import {
 import { isUpdateAvailable } from "../helpers/cli";
 
 vi.mock("process");
+vi.mock("@cloudflare/cli/interactive");
 
 function promisify<T>(value: T) {
 	return new Promise<T>((res) => res(value));
 }
 
+beforeEach(() => {
+	// we mock `spinner` to remove noisy logs from the test runs
+	vi.mocked(spinner).mockImplementation(() => ({
+		start() {},
+		update() {},
+		stop() {},
+	}));
+});
```

Comment:
This works but I note that in the Wrangler tests we mock out `logUpdate` instead:

https://github.com/cloudflare/workers-sdk/blob/dae30015c646502819d79bf8b8ae032c4aa0669d/packages/wrangler/src/__tests__/jest.setup.ts#L53-L59

Reply:
yes, I tried doing that too, but `log-update` is a dependency of the [cli package](https://github.com/cloudflare/workers-sdk/tree/d70f44a1716f0a4a87a7a0732bb61079da036dcc/packages/cli) not `create-cloudflare` itself, so vitest was complaining that it could not find the dependency, I didn't try but I imagine I could add that as a `devDependency` of `create-cloudflare`, but it felt simpler, cleaner and more correct to mock the `cli` package instead.

I am totally fine revisiting this and mock `log-update` instead if that's preferred 🙂